content/pt-br/docs/tasks/access-application-cluster/access-cluster-services.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Acessando serviços em execução em clusters
+content_type: task
+weight: 140
+---
+
+<!-- overview -->
+Esta página mostra como se conectar a serviços em execução no cluster Kubernetes.
+
+{{% heading "pré-requisitos" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Acessando serviços em execução no cluster
+
+No Kubernetes, [nós](/docs/concepts/architecture/nodes/), [pods](/docs/concepts/workloads/pods/) e [serviços](/docs/concepts/services-networking/service/) têm todos seus próprios IPs. Em muitos casos, os IPs dos nós, dos pods e alguns dos IPs de serviço em um cluster não serão
+roteáveis, portanto, não estarão acessíveis a partir de uma máquina fora do cluster, como seu computador de mesa.
+
+## Maneiras de se conectar
+
+Você tem várias opções para se conectar a nós, pods e serviços de fora do cluster:
+
+  - Acesse serviços através de IPs públicos.
+    - Use um serviço com tipo `NodePort` ou `LoadBalancer` para tornar o serviço acessível fora do cluster. Consulte a documentação de [serviços]((/docs/concepts/services-networking/service/)) e
+[kubectl expose](/docs/reference/generated/kubectl/kubectl-commands/#expose).
+    - Dependendo do ambiente do cluster, isso pode expor o serviço apenas para a rede corporativa, ou pode expô-lo para a Internet. Pense se o serviço que está sendo exposto é seguro. Ele faz sua própria autenticação?
+    - Coloque pods atrás de serviços. Para acessar um pod específico de um conjunto de réplicas, como para depurar, coloque um rótulo exclusivo no pod e crie um novo serviço que selecione este rótulo.
```

Comment:
"Pod" instead of "pod". 

We standardized use of Pod with a capital letter. Check below